### PR TITLE
Add configuration for solomon exporter: specify host and some instance tags

### PR DESCRIPTION
**Pull request by achulkov2**
When the solomon_exporter section is not configured metrics are exported without any host-related labels, which isn't very convenient.

Tested locally that the sensors from `curl localhost:<monitoring_port>/solomon/all` are now enriched with the added attributes.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -33,7 +33,7 @@ else
 DOCKER_BUILD_ARGS += --build-arg VERSION="$(OPERATOR_TAG)"
 endif
 DOCKER_BUILD_ARGS += --build-arg REVISION="$(shell git rev-parse HEAD)"
-DOCKER_BUILD_ARGS += --build-arg BUILD_DATE="$(shell date --rfc-3339=seconds)"
+DOCKER_BUILD_ARGS += --build-arg BUILD_DATE="$(shell date -Iseconds)"
```
Comment (zlobober) [inline Makefile:36]: Let's move this to a separate PR.

File: pkg/ytconfig/canondata/TestGetControllerAgentsConfig/test.canondata [hunk 2]
```diff
@@ -4,6 +4,12 @@
         "enable_ipv6"=%false;
         retries=1000;
     };
+    "solomon_exporter"={
+        host="{POD_SHORT_HOSTNAME}";
+        "instance_tags"={
+            "k8s_pod_name"="{K8S_POD_NAME}";
```
Comment (zlobober) [inline pkg/ytconfig/canondata/TestGetControllerAgentsConfig/test.canondata:10]: I'd say that "k8s_" is excessive here, maybe just "pod_name", or even "pod"? I'd prefer having the most often used labels to be as concise as possible. Also, we have "host", not "host_name"/"hostname".

File: Makefile (hunk 1 above)
Comment (achulkov2) [reply Makefile:36]: Ok, moved here: https://github.com/ytsaurus/yt-k8s-operator/pull/260.

File: pkg/ytconfig/canondata/TestGetControllerAgentsConfig/test.canondata (hunk 2 above)
Comment (achulkov2) [reply pkg/ytconfig/canondata/TestGetControllerAgentsConfig/test.canondata]: -> pod

**Review by l0kix2 — APPROVED** [review]

**Comment by l0kix2** [comment]
LGTM, let's merge if @zlobober ok with added test.

**Review by zlobober — APPROVED** [review]
